src/algebraic_topology/simplex_category.lean

```diff
@@ -494,6 +494,246 @@ end
 lemma le_of_epi {n m : ℕ} {f : [n] ⟶ [m]} : epi f → (m ≤ n) :=
 len_le_of_epi
 
+instance {n : ℕ} {i : fin (n+2)} : mono (δ i) :=
+begin
+  rw mono_iff_injective,
+  exact fin.succ_above_right_injective,
+end
+
+instance {n : ℕ} {i : fin (n+1)} : epi (σ i) :=
+begin
+  rw epi_iff_surjective,
+  intro b,
+  simp only [σ, mk_hom, hom.to_order_hom_mk, order_hom.coe_fun_mk],
+  by_cases b ≤ i,
+  { use b,
+    rw fin.pred_above_below i b (by simpa only [fin.coe_eq_cast_succ] using h),
+    simp only [fin.coe_eq_cast_succ, fin.cast_pred_cast_succ], },
+  { use b.succ,
+    rw [fin.pred_above_above i b.succ _, fin.pred_succ],
+    rw not_le at h,
+    rw fin.lt_iff_coe_lt_coe at h ⊢,
+    simpa only [fin.coe_succ, fin.coe_cast_succ] using nat.lt.step h, }
+end
+
+/-- A bijective map in `simplex_category` is an isomorphism. -/
+@[simps]
+noncomputable def iso_of_bijective {x y : simplex_category.{u}} {f : x ⟶ y}
```

Comment:
Do we have an instance of `reflects_isomorphisms` for `forget simplex_category`? If not, it would be worthwhile to add, and obtain this `def` as a corollary.

Reply:
Thanks for the suggestion. In the last commit, I have obtained this instance and deduced `is_iso_of_bijective` from it. (About what I plan to do with this PR, these results are used in my proof of the Dold-Kan equivalence between chain_complexes and simplicial objects in a (pseudo)abelian category.)